doc/guides/Developer-s Guide/plugin-development/event-notifications.md

```diff
@@ -87,9 +87,11 @@ A notification for topic `connect` is sent every time a new connection to a peer
 
 ```json
 {
```

Comment:
We should talk about the form of all these notifications (i.e. that notification X will contain an object X) in the general section of this doc.

Should we remove PLUGINS.md too, as a previous commit?  It's confusing to have both!

Reply:
Agree, it is redundant but we can leave it till I am not working on documentation consolidation. I will work on the logical placement of all these documents at that point anyways.